components/cli/pkg/commands/image/test.go

```diff
@@ -275,7 +274,7 @@ func startTestCellInstance(cli cli.Cli, imageDir string, instanceName string, ru
 	} else {
 		var balModule string
 
-		if instanceName != "" {
+		if instanceName != "" && instanceName != "test" {
```

Comment:
Don't we need to inform the user that the name "test" can't use as an instance name?

Reply:
No since this is only used for the name of the ballerina module to extract and run the tests which is an implementation detail. The user still can create an intance named "test".

Reply:
We would have similar issues for all keywords and already existing package names in ballerina right ? Not just `test`

Reply:
Yes, that should be handled by Ballerina. Created an issue to track this.
https://github.com/ballerina-platform/ballerina-lang/issues/19997